service-statistics-common/src/main/java/fi/nls/oskari/control/statistics/data/RegionValue.java

```diff
@@ -0,0 +1,20 @@
+package fi.nls.oskari.control.statistics.data;
+
+public class RegionValue {
+    private String region;
```

Comment:
These fields could be marked as `final` even though they already are effectively `final`.